docs/asciidoc/static/include/pluginbody.asciidoc

```diff
@@ -0,0 +1,1145 @@
+:branch:          1.5
+:ls_version:		  1.5.0.beta1
+
+= How to write a Logstash {plugintype} plugin
+
+To develop a new {plugintype} for Logstash, you build a self-contained Ruby gem
+that lives in its own GitHub repository. You can use the example {plugintype}
```

Comment:
(as discussed)
To develop a new {plugintype} for Logstash, you build a self-contained Ruby gem whose source code lives in its own GitHub repository. The Ruby gem may then be hosted and shared on RubyGems.org (or perhaps a private gem hosting service of your choosing).